RocketChat/Rocket.Chat.ReactNative
# feat: play audio in sequence

## Proposed changes


## Issue(s)	


## How to test or reproduce


- Test audios in sequence
- Pls, see the video at # Screenshots to see the edge cases that I was able to avoid

## Screenshots

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/93720aa7-a2bc-4cdb-97bf-2fadb114f633


## Types of changes




## Checklist



## Further comments

[TC-1095]

[TC-1095]: https://rocketchat.atlassian.net/browse/TC-1095?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
